Add Snowflake JDBC error troubleshooting section (#62929)
metabase/metabase: docs/troubleshooting-guide/db-connection.md

   SELECT 1
   ```

## Snowflake error when running via JAR file

If you're connecting to Snowflake and encounter an error like, `JDBC driver internal error: exception creating result` and are running from the JAR file, add the following option to your java command:

```
java --add-opens java.base/java.nio=ALL-UNNAMED -jar metabase.jar
```

> See [the documentation on running the Metabase jar file](https://www.metabase.com/docs/latest/installation-and-operation/running-the-metabase-jar-file) for more details.

Reviewer: ```suggestion
See the docs on [running the Metabase jar file](../installation-and-operation/running-the-metabase-jar-file.md).
```
Links should be relative (not pinned to latest).
